assets/stylesheets/bootstrap/mixins/_opacity.scss

```diff
@@ -1,7 +1,7 @@
 // Opacity
 
 @mixin opacity($opacity) {
-  $opacity-ie: ($opacity * 100);
+  $opacity-ie: ($opacity * 100);  // IE8 filter
```

Comment:
Let's forget about IE8 in comments please.  I'm ok with leaving $opacity-ie however maybe that should also go.